tts-finetuning-cvtool.ipynb

```diff
@@ -0,0 +1,902 @@
+   "outputs": [],
+   "source": [
+    "merged_ds = dataset_merge(os.environ[\"original_data_json\"], os.environ[\"finetune_data_json\"])\n",
+    "\n",
+    "os.environ[\"merged_data_json\"] = f\"{DATA_DIR}/{finetune_data_name}/merged_train.json\"\n",
+    "json_writer(os.environ[\"merged_data_json\"], merged_ds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Getting Pitch Statistics\n",
+    "\n",
+    "Training Fastpitch requires you to set 2 values for pitch extraction:\n",
+    "  - `avg`: The average used to normalize the pitch\n",
+    "  - `std`: The std deviation used to normalize the pitch\n",
+    "\n",
+    "We can compute pitch for the training data using [`scripts/dataset_processing/tts/extract_sup_data.py`](https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/extract_sup_data.py) and extract pitch statistics using the NeMo script [`scripts/dataset_processing/tts/compute_speaker_stats.py`](https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/compute_speaker_stats.py), so let's download the scripts and use it to get `pitch_mean` and `pitch_std`. Let's download the required files."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/scripts/dataset_processing/tts/extract_sup_data.py\n",
+    "! mkdir -p ljspeech && cd ljspeech \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/scripts/dataset_processing/tts/ljspeech/ds_conf/ds_for_fastpitch_align.yaml \\\n",
+    "&& cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we will extract the pitch supplementary data using `extract_sup_data.py` file. This file works with a yaml config file `ds_for_fastpitch_align`, which we downloaded above. To make this work for your dataset simply change the `manifest_path` to your manifest path. The argument `sup_data_path` determines where the supplementary data is stored."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sup_data_path = f'{finetune_data_path}/sup_data_path'\n",
+    "pitch_stats_path = f'{finetune_data_path}/pitch_stats.json'\n",
+    "\n",
+    "! python extract_sup_data.py --config-path \"ljspeech\" manifest_filepath={os.environ[\"merged_data_json\"]} sup_data_path={sup_data_path}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setting the `pitch_fmean` and `pitch_fmax` based on the results from the cell above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please set the pitch_mean and pitch_std values based on\n",
+    "# the output from the above pitch_stats task.\n",
+    "pitch_mean = FIX_ME\n",
+    "pitch_std = FIX_ME\n",
```

Comment:
Added a string parser logic to get the values from commandline output. Since we are executing a python file, it is difficult to get the returned value in a variable.